quiche/src/frame.rs

```diff
@@ -987,20 +989,25 @@ impl Frame {
                 error_code, reason, ..
             } => QuicFrame::ConnectionClose {
                 error_space: Some(ErrorSpace::TransportError),
-                error_code: Some(*error_code),
-                error_code_value: None, // raw error is no different for us
+                // TODO: determine if connection is established so we can log
+                // CryptoErrors
+                error_code: Some(ConnectionCloseErrorCode::Numeric(*error_code)),
```

Comment:
I think what we want to do instead convert all the known error codes to actual qlog transport or crypto errors, rather than a numeric one